api.py

```diff
@@ -1,21 +1,24 @@
+import pickle
 from fastapi import FastAPI
 from pydantic import BaseModel
-import pickle
 import pandas as pd
 
 app = FastAPI()
 
-class ScoringItem(BaseModel): 
-    YearsAtCompany: float 
-    EmployeeSatisfaction: float 
-    Position:str
-    Salary: int
 
-with open('model/model.pkl', 'rb') as f: 
+class ScoringItem(BaseModel):
+    sepal_length: float
+    sepal_width: float
+    petal_length: float
+    petal_width: float
+
+
+with open("model/model.pkl", "rb") as f:
     model = pickle.load(f)
 
-@app.post('/')
-async def scoring_endpoint(item:ScoringItem): 
+
+@app.post("/")
+async def scoring_endpoint(item: ScoringItem):
```

Comment:
should we configure some validation and error handling to set the correct status code for specific cases? At this moment, we can use the following ones:

* 500 Internal Server Error: A generic error message indicating that something went wrong on the server's end, but the server couldn't provide specific details.
* 400 Bad Request: The server could not understand the request due to invalid syntax or missing parameters.

Eventually we will have more cases